assets/css/_route_ladder.scss

```diff
@@ -49,16 +49,24 @@
 
   .card-body {
     display: inline-grid;
-    padding: 0;
+    grid-template-columns: 1fr 1fr auto 1fr 1fr;
+    padding: 6px 8px;
     align-items: center;
   }
 
-  .c-new-route-ladder__fake-dropdown {
+  .c-route-ladder__dropdown-button {
     height: 36px;
```

Comment:
note/slight issue(from previous PR?): Github won't let me comment on the unchanged lines, but I don't think `36px` is the proper unit to use here. If the user uses a different root font size than `16px`, this doesn't actually end up looking correct anymore, I think it'd be more appropriate to use `height: 2.25rem; width: 2.25rem;` here, and even better to use `aspect-ratio: 1/1;` as noted in my other comment.

Reply:
Defer to this thread though https://github.com/mbta/skate/pull/2623#discussion_r1617224651